Content.Shared/Light/EntitySystems/LightReplacerSystem.cs

```diff
@@ -210,7 +198,7 @@ public bool TryInsertBulb(EntityUid replacerUid, EntityUid bulbUid, EntityUid? u
     ///     Returns true if storage contained at least one light bulb
     ///     which was successfully inserted inside light replacer
     /// </returns>
-    public bool TryInsertBulbsFromStorage(EntityUid replacerUid, EntityUid storageUid, EntityUid? userUid = null,
+    private bool TryInsertBulbsFromStorage(EntityUid replacerUid, EntityUid storageUid, EntityUid? userUid = null,
```

Comment:
For these, I think leaving them as public is fine. It's more nuanced but there may be something in the future that might use this as part of a call to the system.